pytest.ini

```diff
@@ -1,5 +1,5 @@
 [pytest]
 DJANGO_SETTINGS_MODULE=example.settings.test
 filterwarnings =
-    error::DeprecationWarning
-    error::PendingDeprecationWarning
+    ignore::DeprecationWarning
```

Comment:
those deprecation warning errors are here for a reason so when new updates of dependencies are automatically added they won't be automatically committed if the introduce deprecation warnings.

It is actually possible to ignore test on a test level see https://docs.pytest.org/en/latest/warnings.html and we need to redesign the deprecation warning so it only occurs in cases where a user is affected. See my comments below.